docs/self-managed/setup/upgrade.md

```diff
@@ -2,130 +2,61 @@
 id: upgrade
 title: "Camunda 8 Helm upgrade"
 sidebar_label: "Upgrade"
-description: "To upgrade to a more recent version of the Camunda Helm charts, there are certain things you need to keep in mind."
+description: "Upgrade to a more recent version of the Camunda Helm charts."
```

Comment:
I LOVE that you've tidied this up something significantly cleaner. This is a nitpick, but I try to shoot for ~150 characters for SEO purposes 👍